sites/README.md

```diff
@@ -0,0 +1,293 @@
+# Valid properties
+
+List of possible values that can be used
+
+## name
+**(Mandatory)**
+
+**Type:** String
+
+This property will be used mainly for all the Kubernetes metadata name, it will also be used to set some properties as a default value if they have not been defined as tlsSecretName.
+
+Example:
+```
+name: canonical-com
+```
+
+## domain
+**(Mandatory)**
+
+**Type:** String
+
+This property define the project domain, it will be used for the Service, Deployment and Ingress objects.
+
+Example:
+```
+domain: canonical.com
+```
+
+## noIngress
+
+**Type:** Boolean
+
+**Default:** False
+
+When this property is set to true, the Ingress object will be empty.
+
+Example:
+```
+noIngress: true
+```
+
+## container
+**(Mandatory)**
+
+This property is for container related configurations. Here you can set the [image](#image) and the [tag](#tag).
+
+### image
+**(Mandatory)**
+
+**Type:** String
+
+Set the Docker image you want to use.
+
+Example:
+```
+container:
+    image: prod-comms.docker-registry.canonical.com/canonical.com
+```
+
+### tag
+
+**Type:** String
+
+**Default:** latest
+
+You can specify the tag you want to use with the image.
+
+Example:
+```
+container:
+    image: prod-comms.docker-registry.canonical.com/canonical.com
+    tag: a264efb326485
+```
+
+## wwwPrefixHost
+
+**Type:** Boolean
+
+**Default:** True
+
+When True it will include an extra host with the www prefix to the Ingress object.
+
+Example:
+```
+wwwPrefixHost: false
+```
+
+## tlsSecretName
+
+**Type:** String
+
+**Default:** {name}-tls
+
+Here you can define the name of the TLS secret for the domain. This name will be used for the www prefix host if `wwwPrefixHost` is True.
+
+Example:
+```
+tlsSecretName: cloud-init-org-tls
+```
+
+## useSquid
+
+**Type:** Boolean
+
+**Default:** False
+
+This will use our ConfigMap object "proxy-config", you can take a look to this object here: `configmaps/proxy-config.yaml`
+
+Example:
+```
+useSquid: True
+```
+
+## readinessPath
```

Comment:
Again, could we remove this option and make it always use `/status/_check`? Then if this doesn't work for any site we have, we should just fix the sites to make sure it does work. It should be pretty trivial to do this in all cases.

Reply:
I have left it for now as we agreed

Reply:
I filed https://github.com/canonical-web-and-design/deployment-configs/issues/281 for removing this option down the line.

Reply:
Perfect!